### DISCUSSION TITLE
Hide the sorting icon

### DISCUSSION THREAD

**Discussion by livevsonline** [TanStack/table#4636] [Q&A] [answered]
Hi, right now i get a blue sorting icon

I don't see any way to hide this?

Basically i don't want anything to show at all when sorting, no arrow up/down no nothing, but i want to keep the sorting functionality?

Regards

**Comment by barbinbrad** [comment, marked as answer]
Can you post your code? I'll be happy to take a look.

**Reply by livevsonline** [reply]
> Can you post your code? I'll be happy to take a look.

I figured it out, it was myself who added it.. doohh...
